### PR TITLE
Deprecating tools that we no longer want to support

### PR DESCRIPTION
**Pull request by eitanbanks**
@ktibbett: deprecating these classes as requested.
I'll also add an issue here to remove them in a few months.

### CODE REVIEW

**Comment by ktibbett** [comment]
:+1:

File: src/tests/java/picard/analysis/CollectWgsMetricsFromQuerySortedTest.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  *
  * @author Eric Banks
  */
-
+@Deprecated
```
Comment (yfarjoun) [inline src/tests/java/picard/analysis/CollectWgsMetricsFromQuerySortedTest.java]: please comment what class to use instead.

File: src/tests/java/picard/analysis/CollectWgsMetricsFromSampledSitesTest.java [hunk 2]
```diff
@@ -40,6 +40,7 @@
 /**
  * Tests CollectWgsMetricsFromSampledSites
  */
+@Deprecated
```
Comment (yfarjoun) [inline src/tests/java/picard/analysis/CollectWgsMetricsFromSampledSitesTest.java]: same.

File: src/tests/java/picard/analysis/CollectWgsMetricsFromQuerySortedTest.java (hunk 1 above)
Comment (yfarjoun) [inline src/tests/java/picard/analysis/CollectWgsMetricsFromQuerySortedTest.java]: sorry, I mean on the main class, not the test.

**Comment by eitanbanks** [comment]
Added a note about which tool to use instead.  Unless I hear any objections, I will merge this later on today.

**Pull request merged**